include/arch/arm/aarch32/cortex_a_r/cpu.h

```diff
@@ -48,6 +48,7 @@
 #define DFSR_AXI_SLAVE_MASK	(1 << 12)
 
 /* Armv8-R AArch32 architecture profile */
+#define VBAR_Msk		(0xFFFFFFE0U)
```

Comment:
It would be nice if this is named `VBAR_MASK` like the rest in this file (e.g. `DFSR_WRITE_MASK`, `MPIDR_AFFLVL_MASK`, ...), but I leave this up to the maintainer.

Reply:
You're right I tried to make it consistent with the CONFIG_CPU_CORTEX_M_HAS_VTOR case, but VBAR_MASK is better